completions/bash/brew

```diff
@@ -104,728 +104,2181 @@ __brew_complete_commands() {
   COMPREPLY=($(compgen -W "$cmds" -- "$cur"))
 }
 
-_brew_analytics() {
-  case "$COMP_CWORD" in
-    2)  __brewcomp "off on regenerate-uuid state" ;;
-  esac
-}
-
-_brew_bottle() {
+_brew_abv() {
   local cur="${COMP_WORDS[COMP_CWORD]}"
   case "$cur" in
     -*)
       __brewcomp "
-        --debug
-        --force-core-tap
-        --help
-        --json
-        --keep-old
-        --merge
-        --no-commit
-        --no-rebuild
-        --or-later
-        --root-url
-        --skip-relocation
-        --verbose
-        --write
-        "
+      --all
+      --analytics
+      --cask
+      --category
+      --days
+      --debug
+      --formula
+      --github
+      --help
+      --installed
+      --json
+      --quiet
+      --verbose
+      --verbose
```

Comment:
This probably shouldn't get listed twice here.

Reply:
Good catch, thanks!